### PR TITLE
updated remember a device page

### PR DESCRIPTION
**Pull request by tannerabread**
_Issue #, if available:_

_Description of changes:_
Update Auth/Remember a device page including code formatting for js/react-native and outdated screenshots for cognito console

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by abdallahshaban557 — CHANGES_REQUESTED** [review]

File: src/fragments/lib/auth/common/device_features/common.mdx [hunk 1]
```diff
@@ -18,13 +18,13 @@ amplify auth console
     User Pool
 ```
 
-When the console opens, **click on Devices** from the left navigation menu, which will render the following page allowing you to configure your preference for remembering a user's device.
+When the console opens, scroll down to the **Device Tracking** section and click the `Edit` button. This will render the following page allowing you to configure your preference for remembering a user's device.
```
[the remark this answers is not in the document]
Comment (tannerabread) [reply src/fragments/lib/auth/common/device_features/common.mdx]: done

**Review by abdallahshaban557 — APPROVED** [review]
LGTM!

**Review by nadetastic — APPROVED** [review]
LGTM

**Review by cwomack — APPROVED** [review]
Looks good!

**Pull request merged**